test/demo/raiseIntent.test.ts

```diff
@@ -76,35 +75,42 @@ describe('Intent listeners and raising intents', () => {
                 });
 
                 test('When calling raiseIntent from another app the listener is triggered exactly once with the correct context', async () => {
-                    await fdc3Remote.raiseIntent(testManagerIdentity, validPayload.intent, validPayload.context, testAppIdentity.appId);
+                    await fdc3Remote.raiseIntent(testManagerIdentity, validPayload.intent, validPayload.context, testAppIdentity.name);
 
                     const receivedContexts = await listener.getReceivedContexts();
                     expect(receivedContexts).toEqual([validPayload.context]);
                 });
                 test('When calling unsubscribe from the intent listener, then calling raiseIntent from another app, it times out', async () => {
                     await listener.unsubscribe();
-                    const resultPromise = fdc3Remote.raiseIntent(testManagerIdentity, validPayload.intent, validPayload.context, testAppIdentity.appId);
+                    const resultPromise = fdc3Remote.raiseIntent(testManagerIdentity, validPayload.intent, validPayload.context, testAppIdentity.name);
 
-                    await expect(resultPromise).rejects.toThrowError(`Timeout waiting for intent listener to be added. intent = ${validPayload.intent}`);
-                }, INTENT_LISTENER_TIMEOUT + 500);
+                    await expect(resultPromise).toThrowFDC3Error(
+                        OpenError.AppTimeout,
+                        `Timeout waiting for intent listener to be added. intent = ${validPayload.intent}`
+                    );
+                }, Timeouts.ADD_INTENT_LISTENER + 500);
             });
 
-            describe('When the target is *not* registered to accept the raised intent', () => {
+            describe('When the target is not in the directory', () => {
                 test('When calling raiseIntent the promise rejects with an FDC3Error', async () => {
-                    const resultPromise = fdc3Remote.raiseIntent(testManagerIdentity, invalidPayload.intent, invalidPayload.context, testAppIdentity.appId);
-                    // TODO: decide if this is this the error message we want when sending a targeted intent
-                    await expect(resultPromise).rejects.toThrowError(/No applications available to handle this intent/);
-                    await expect(resultPromise).rejects.toHaveProperty('name', 'FDC3Error');
-                    await expect(resultPromise).rejects.toHaveProperty('code', ResolveError.NoAppsFound);
+                    // TODO? Does this test make sense here? 'When the target is running' but then 'When the app doesn't exist'? [SERVICE-479 will change this]
```

Comment:
Would change to `TODO:`. Or if I'm understanding the test and SERVICE-479 correctly, it would make sense to move this and get rid of the TODO

Reply:
Yes 479 is going to affect these tests quite a bit - Changing to `TODO:` by now